Makefile

```diff
@@ -176,3 +179,34 @@ else
 	rm -rf nix-remote-builder
 endif
 endif
+
+.PHONY: release-dry-run
+release-dry-run:
+	docker run \
+		-it \
```

Comment:
```suggestion
```
github actions cannot run with `-it`
with error
`the input device is not a TTY`